packages/camera/camera_android_camerax/CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributing to camera\_android\_camerax
```

Comment:
Should we link to this page from the README?

Reply:
Yea i think this would be helpful to other contributors.